tutorials/using-cloud-vpn-with-alibaba-with-redundancy

```diff
@@ -0,0 +1,950 @@
+point of view of the GCP VPC network, so "on-premises gateway" refers to the
+gateway that's connecting _to_ GCP.
+-  **External IP address** or **GCP peer address**—a single static IP address
+within a GCP project that exists at the edge of the GCP network.
+-  **Static routing**—Manually specifying the route to subnets on the GCP
+side and to the on-premises side of the VPN gateway.
+
+**Alibaba Terminology**
+-  **Alibaba Cloud VPC**-A private network established in Alibaba Cloud. VPCs 
+are logically isolated from other virtual networks in Alibaba Cloud. VPCs allow 
+you to launch and use Alibaba Cloud resources in your VPC.
+-  **Alibaba Cloud VSwitch**-A VSwitch is a basic network device of a VPC and 
+used to connect different cloud product instances. When creating a cloud product 
+instance in a VPC, you must specify the VSwitch that the instance is located.
+-  **Alibaba Cloud Zone**-Zones are physical areas with independent power grids 
+and networks in one region. Alibaba recommends creating different VSwitches in 
+different zones to achieve disaster Recovery.
+-  **Alibaba Cloud VPN Gateway**-The VPN gateway is the IPsec VPN gateway created 
+on the Alibaba Cloud side. One VPN gateway can have multiple VPN connections.
+-  **Alibaba Cloud Customer Gateway**-The customer gateway is the VPN service 
+deployed in the on-premises data Center or, in this case, the GCP Cloud VPN gateway. 
+By creating a customer gateway, you can register the VPN information to the cloud, 
+and then create a VPN connection to connect the VPN gateway and the customer gateway.
+-  **Alibaba Cloud VRouter**-A VRouter is a hub in the VPC that connects all 
+VSwitches in the VPC and serves as a gateway device that connects the VPC to 
+other networks. VRouter routes the network traffic according to the configurations 
+of route entries.
+-  **Alibaba Cloud Route Entry**-A route entry specifies the next hop address for the 
+network traffic destined to a CIDR block. It has two types of entries: system route 
+entry and custom route entry.
+-  **Alibaba Cloud Route Table**-A route table is a list of route entries in a VRouter.
+
+## Topology
+
+Cloud VPN supports the following topology with Alibaba Cloud VPN Gateway:
+
+-  Redundant IPsec VPN tunnel configuration using static routing.
+
+<p><img align="center" src="topology-redundancy.PNG" height="200" /></p>
+
+For detailed topology information, see the following resources:
+
+-  For basic VPN topologies, see 
+[Cloud VPN Overview](https://cloud.google.com/vpn/docs/concepts/overview).
+-  For redundant topologies,  the
+[Cloud VPN documentation on redundant and high-throughput VPNs](https://cloud.google.com/vpn/docs/concepts/redundant-vpns).
+
+Disclaimer: At time of this writing, IPsec VPN tunnel configuration using dynamic routing between Cloud VPN and Alibaba Cloud VPN Gateway is not supported. 
+
+## Product environment
+
+The on-premise VPN gateway used in this guide is as follows:
```

Comment:
change: The on-premise VPN gateway used in this guide is as follows:
to: The on-premise VPN gateways used in this guide are: